internal/provider/data_source_gitlab_groups_test.go

```diff
@@ -0,0 +1,56 @@
+//go:build acceptance
+// +build acceptance
+
+package provider
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccDataSourceGitlabGroups_basic(t *testing.T) {
+	prefixFoo := "acctest-group-foo"
+	groupsFoo := testAccCreateGroupsWithPrefix(t, 2, prefixFoo)
+
+	prefixLotsOf := "acctest-group-lotsof"
+	testAccCreateGroupsWithPrefix(t, 42, prefixLotsOf)
+
+	resource.ParallelTest(t, resource.TestCase{
+		ProviderFactories: providerFactories,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDataSourceGitlabGroupsConfigSearchSort(prefixFoo),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("data.gitlab_groups.foos", "groups.#", "2"),
+					resource.TestCheckResourceAttr("data.gitlab_groups.foos", "groups.0.name", groupsFoo[0].Name),
```

Comment:
Could you validate the group attributes here as well?

Reply:
I committed validating the group attributes.

Reply:
Thanks, it looks good. 👍 Could you make sure the CI workflow is passing? There is a guide in the CONTRIBUTING.md, but tl;dr you can run `make reviewable` from the console.

Reply:
I committed the change resulting from "make reviewable".
I also run "make testacc", but cannot run "make testacc SERVICE=gitlab-ee".